main.rb

```diff
@@ -0,0 +1,39 @@
+require './app'
+
+def main
+  @app = App.new
+end
```

Comment:
When I run the script `ruby main.rb` nothing happen.
Please correct it to follow requirements :[OOP school library: add basic UI](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md)